src/components/Header/Wallet/Details.tsx

```diff
@@ -67,7 +67,7 @@ export default function Details(): ReactElement {
               <AddToken
                 address={oceanTokenMetadata?.address}
                 symbol={oceanTokenMetadata?.symbol}
-                logo="https://raw.githubusercontent.com/oceanprotocol/art/main/logo/token.png"
+                logo="ocean"
```

Comment:
up here, the logo is supposed to be for the datatoken, as this action adds the datatoken to the wallet. Just like with pricing screen, this raises question what exact logo datatokens should have, as this was never consciously decided. I would see 3 options:

1. Giving datatokens the ocean logo, but in violet, was a quick solution when building v3, but it also reflects that they are still OCEAN-branded tokens in a way. From that perspective we could go with violet OCEAN logo for all datatokens, regardless of base token.

2. Current implementation where base token logo is made violet works also well if we are fine with them being branded differently. But mechanics-wise the datatoken as access token is not bound or derived from the base tokens so argument could be made this could be confusing to users.

4. We could come up with a new logo, for all datatokens. Of course would take longest as design work has to be done first.

Reply:
3 would be the best , but i would go with 1. for now.

Reply:
So, are we going with solution 1 for the moment? Can I go ahead and change it?

Reply:
go for it

Reply:
Done